client/common/sass/_typography.sass

```diff
@@ -48,3 +48,28 @@ body
 
 h1, h2, h3, h4, h5, h6
 	margin: 1rem 0
+
+p
+	font-size: $font-size-p
+	line-height: 1.33
+	font-weight: 400
+
+	+desktop-up
+		font-size: $font-size-p-desktop
+
+a
+	font-size: $font-size-small-p
+	font-weight: 400
+	line-height: 1.43
+	color: $black
+
+	.rich-text &
+		font-size: inherit
+		line-height: inherit
+
+	&:not(.btn):focus
+		border: 3px solid
+		border-color: $black
+
+	&:not(.btn):not(.text-link--card):not(.incident-index__download-item):hover
+		background-color: $accent-color
```

Comment:
I removed this generic `p` and `a` styles because they were conflicting too much. Wherever the common link style is needed, we can add `text-link` class. Is there any other reason to bring this back?